osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@AndreasVIRES I see your point: you would like all vehicles to have the same informations. However for non-ego vehicles this information is never required. We should rather consider dropping relative_ego_origin generally. It seems to cause irritations and can in principal be handled by the environment model. I will open an issue.